lighthouse-core/gather/gatherers/dobetterweb/domstats.js

```diff
@@ -86,8 +86,8 @@ function elementPathInDOM(element) {
 /* istanbul ignore next */
 function getDOMStats(element, deep = true) {
   let deepestElement = null;
-  let maxDepth = 0;
-  let maxWidth = 0;
+  let maxDepth = -1;
```

Comment:
`maxDepth` doesn't need to change (`<body>` counts as depth 1 which is > 0), but it seemed weird not to change it as well :)